ISICbalance/ViewModels/AccountViewModel.swift

```diff
@@ -28,22 +28,19 @@ extension AccountViewModelingActions where Self: AccountViewModeling {
 
 final class AccountViewModel: BaseViewModel, AccountViewModeling, AccountViewModelingActions {
     typealias Dependencies = HasKeychainManager
-    private let dependencies: Dependencies // TODO: není potřeba
 
     let username: MutableProperty<String>
     let password: MutableProperty<String>
-    private var validationSignal: Property<Bool> // TODO: není potřeba
-    private var validationErrors: Property<[LoginValidation]> // TODO: není potřeba
+
     let loginAction: Action<(),(),LoginError>
 
     // MARK: - Initialization
     init(dependencies: Dependencies) {
-        self.dependencies = dependencies
 
         username = MutableProperty("")
         password = MutableProperty("")
 
-        validationErrors = username.combineLatest(with: password).map { username, password in // TODO: používat Producer.combineLatest([...])
+        var validationErrors: Property<[LoginValidation]> = username.combineLatest(with: password).map { username, password in // TODO: používat Producer.combineLatest([...])
```

Comment:
❔ proč `var`?

Reply:
Protože je to ještě v rozpracované fázi a chci se na to zeptat v pondělí Honzy. Nevím moc jak myslel ten komentář.